presentation-api/src/main/java/org/cytoscape/view/presentation/property/AbstractVisualLexicon.java

```diff
@@ -0,0 +1,200 @@
+package org.cytoscape.view.presentation.property;
```

Comment:
I pulled some code that was in BasicVisualLexicon up into this abstract class so it could be reused by BasicTableVisualLexicon.